drivers/storage/portworx/feature.go

```diff
@@ -0,0 +1,18 @@
+package portworx
+
+import (
+	"strconv"
+)
+
+// Feature is the enum type for different features
+type Feature string
+
+const (
+	// CSI is used to indicate CSI feature
+	CSI Feature = "CSI"
```

Comment:
prefix all feature constants with "Feature".

Since the pkg name is portworx, the usages will look like portworx.CSI. So it isn't intuitive this is a feature flag.